lmp/bb-config.sh

```diff
@@ -8,7 +8,7 @@ OSTREE_BRANCHNAME="${OSTREE_BRANCHNAME-lmp-localdev}"
 SOTA_CLIENT="${SOTA_CLIENT-aktualizr}"
 AKLITE_TAG="${AKLITE_TAG-promoted}"
 H_BUILD="${H_BUILD-lmp-localdev}"
-LMP_VERSION=$(git --git-dir=.repo/manifests/.git describe --tags 2>/dev/null || echo unknown)
+LMP_VERSION="$(LMP_VERSION-unknown)"
```

Comment:
The same here: `"$(LMP_VERSION:-unknown)"` is better to prevent passing forward empty value.

Reply:
I understand you suggestion but using here `:-` when all of the others only uses `-`  doesn't look very nice